docs/application/native/guides/text-input/tts.md

```diff
@@ -686,6 +719,55 @@ To start, pause, and stop the playback, follow the steps below:
   }
   ```
 
+<a name="control"></a>
+## Playing Mode
+
+It is possible to determine whether to request playback of PCM data synthesized by the TTS service or to receive and play it directly by the TTS client.
+
+To decide between the client-side playback mode and the service-side playback mode, follow the steps below:
+
+- Set the playing mode
+
+  If the application wants to play the PCM data directly instead of playing it that the application requested for synthesis in the TTS service, you can set the playback mode to 'TTS_PLAYING_MODE_BY_CLIENT'. If not set, because the default value is 'TTS_PLAYING_MODE_BY_SERVICE', the TTS service will play it when the TTS client requests to synthesize the text.
+
+  > [!NOTE]
+  > If the playing mode is 'TTS_PLAYING_MODE_BY_SERVICE', you don't need to set callback. It will be played automatically by the TTS service.
+
+  ```cpp
+  void
+  set_playing_mode(tts_h tts)
+  {
+      int ret;
+      int playing_mode = TTS_PLAYING_MODE_BY_CLIENT;
+      ret = tts_set_playing_mode(tts, playing_mode);
+      if (TTS_ERROR_NONE != ret)
+          /* Error handling */
+  }
+  ```
+
+- Receive the synthesized pcm data
+
+  To receive the synthesized pcm data from the TTS service to the TTS client, you can use the following callback.
```

Comment:
```suggestion
  To receive the synthesized pcm data from the TTS service to the TTS client, you can use the following callback:
```

Reply:
Thank you for checking. I've just fixed it.